presto-main/src/main/java/com/facebook/presto/execution/SqlQueryExecution.java

```diff
@@ -206,7 +207,7 @@ private SqlQueryExecution(
             stateMachine.setExpandedQuery(queryAnalysis.getExpandedQuery());
 
             stateMachine.beginColumnAccessPermissionChecking();
-            queryAnalyzer.checkAccessPermissions(analyzerContext, queryAnalysis);
+            checkAccessControlPermissions(queryAnalysis.getAccessControlReferences());
```

Comment:
nit: i think `checkAccessPermissions` is a better name than `checkAccessControlPermissions`

Reply:
Yeah, I agree. Earlier this name was taken by column ACL checks, which got cleaned in my later commits. I should have renamed this back. Will change this.